docs/components/sensors/nearfield.md

```diff
@@ -1,3 +1,20 @@
 # Near Field
 
-{!includes/sip.md!}
+Non-visible component to provide NFC capabilities. For now this component supports the reading and writing of text tags only (if supported by the device).  
+
+In order to read and write text tags, the component must have its ReadMode property set to True or False respectively.  
+
+Note: This component will only work on Screen1 of any App Inventor app.
```

Comment:
> App Inventor 

This should be `Kodular`

Reply:
I edited it as "any app created by app inventor distribution" fine?

Reply:
Since these are Kodular's docs, I would suggest you remove all the references to AI2.